sdk/storage/Azure.Storage.Blobs/src/BlobClient.cs

```diff
@@ -195,7 +195,8 @@ public BlobClient(Uri blobUri, TokenCredential credential, BlobClientOptions opt
         /// class with an identical <see cref="Uri"/> source but the specified
         /// <paramref name="snapshot"/> timestamp.
         ///
-        /// For more information, see <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/creating-a-snapshot-of-a-blob" />.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/creating-a-snapshot-of-a-blob">
```

Comment:
No links should have a hard-coded language, i.e.:

```suggestion
        /// For more information, see <see href="https://docs.microsoft.com/rest/api/storageservices/creating-a-snapshot-of-a-blob">
```

Please remove "en-us/" from this and other links within this PR so that users are taken to the preferred language of their browser/OS.

@scbedd should DocWarden be catching these?

Reply:
I didn't change the original URLs just added the link-text by cross-referencing with other links to the same URL.  I see your point.. I find >2000 URLs with language as part of the URL (although a few are external), so this would be quite a big additional change.

Reply:
A very good point. I've opened #18738 to track.